measurement/measures/radioactivity.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from measurement.base import MeasureBase
+
+
+__all__ = [
+    'Radioactivity'
+]
+
+
+class Radioactivity(MeasureBase):
+    """Radioactivity measurements."""
+
+    STANDARD_UNIT = 'bq'
+    UNITS = {
+        'bq': 1,
+        'ci': 37000000000,
+        'rd': 1000000,
+        'dpm': 0.0166666666666666666666666667,
```

Comment:
Float does have the same length on all systems, so this might end up be more accurate and maybe also more readable.
```suggestion
        'dpm': 1/60,
```